src/coreclr/src/jit/compiler.h

```diff
@@ -554,22 +552,47 @@ class LclVarDsc
     }
 #endif // FEATURE_MULTIREG_ARGS
 
-    bool lvIsHfa() const
+    CorInfoHFAElemType GetLvHfaElemKind() const
     {
-#ifdef FEATURE_HFA
-        return IsHfa(_lvHfaElemKind);
+#ifdef FEATURE_HFA_FIELDS_PRESENT
+        return _lvHfaElemKind;
 #else
-        return false;
-#endif
+        NYI("GetLvHfaElemKind");
```

Comment:
These `NYI` should be `NOWAY`, since they're never going to be implemented, it not that they are "not yet implemented".